apps/lm-prompt-optimization/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[project]
+name = "lm-prompt-optimization"
+version = "0.1.0"
```

Comment:
```suggestion
version = "0.0.0-private"
```

Reply:
We cannot use `0.0.0-private`, because it isn't allowed per python rules (https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers), but I can offer you `0.0.0+private`.

Reply:
Sure, also fine 🙂

Reply:
applied in [8a26c92](https://github.com/hashintel/hash/pull/4568/commits/8a26c92f8b1e0baa99f3f587516b91faa4d806c2)